### PR TITLE
Updated README and adding GitHub templates

### PR DESCRIPTION
**Pull request by jwngr**
This makes things more consistent with our other repos and makes the `README.md` a little easier to navigate.

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+- Go through the [documentation](https://firepad.io/docs/)
+- Try out some [examples](../examples/README.md)
+
+If the official documentation doesn't help, try asking a question on the
+[Firepad Google Group](https://groups.google.com/forum/#!forum/firepad-io) or one of our
+other [official support channels](https://firebase.google.com/support/).
+
+**Please avoid double posting across multiple channels!**
+
+see
```
Comment (mikelehen) [inline .github/CONTRIBUTING.md]: I don't see. :-(

**Comment by mikelehen** [comment]
Generally looks great!

**Comment by jwngr** [comment]
Typo fixed. Back to you!

**Comment by mikelehen** [comment]
Merged. Thanks!

**Pull request merged**